### PR TITLE
Enable CircleCI testing in mewarpx

### PR DESCRIPTION
**Pull request by peterscherpelz**
@kzhu-ME @MKieburtz @nruof whichever of use is most interested in continuous integration, plus @roelof-groenewald , can review, given our choice to do 2 reviews for internal PRs.

The guts of this are a CircleCI config file that builds and installs WarpX in a docker image, installs mewarpx, and then runs `pytest` inside mewarpx. You can go to circleci.com to view the actual test results, and we'll have code coverage at codecov.io though we'll need to figure out accounts for that.

### CODE REVIEW

File: mewarpx/tests/test_package.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""Test basic package aspects"""
+from __future__ import print_function
+
```
Comment (roelof-groenewald) [inline mewarpx/tests/test_package.py]: Do we have or plan to have any python2 compatibility? If not this line can be removed, right?
```suggestion
```

**Review by roelof-groenewald — APPROVED** [review]
Not really a comment on this PR but the version test made me think that we should have a discussion on Monday about how we want to do version increments. For example I think it would be good to have the version reflect the current upstream `development` version since that might force API changes or even physics changes. So basically that would mean we do a version increment every week when we merge development into memaster.

**Review by EtherealMC-Bit — APPROVED** [review]
LGTM

File: mewarpx/tests/test_package.py (hunk 1 above)
Comment (peterscherpelz) [reply mewarpx/tests/test_package.py]: Nope, that was just a copy and paste leftover, good catch!

**Review by nruof — APPROVED** [review]
This looks good

**Pull request merged**